src/main/java/com/kuzmins/dao/TicketDAO.java

```diff
@@ -0,0 +1,110 @@
+package com.kuzmins.dao;
+
+import com.kuzmins.config.ConnectionConfig;
+import com.kuzmins.model.Ticket;
+import com.kuzmins.model.TicketType;
+import com.kuzmins.model.User;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class TicketDAO {
+
+    private final static Connection connection = ConnectionConfig.getConnection();
+
+    public void saveTicket(Ticket ticket) {
+        try {
+            PreparedStatement preparedStatement =
+                    connection.prepareStatement("INSERT INTO tickets (id, ticket_type, creation_date) VALUES (?, ?::ticket_type, ?)");
```

Comment:
perfectly will be to convert all queries (`"INSERT INTO tickets (id, ticket_type, creation_date) VALUES (?, ?::ticket_type, ?)"`) in static variables to create them once with class instead of creating with each time of method call

Reply:
converted

Reply:
added method fetchTicketByIdAndUserId(UUID id, UUID userId)